docs/mediapipe.md

```diff
@@ -191,34 +172,49 @@ Nodes in the MediaPipe graphs can reference both to the models configured in mod
 
 Subconfig file may only contain *model_config_list* section  - in the same format as in [models config file](starting_server.md).
 
-## Using MediaPipe <a name="using-mediapipe"></a>
 
-MediaPipe graphs can use the same KServe Inference API as the models. There are exactly the same calls for running
-the predictions. The request format must match the pipeline definition inputs.
+## Deployment testing <a name="testing"></a>
+### Debug logs
+The simples method to validate the graph execution is to set the Model Server `log_level` do `DEBUG`.
+`docker run --rm -it -v $(pwd):/config openvino/model_server:latest --config_path /config/config.json --log_level DEBUG`
 
-Graphs can be queried for their state using the calls [GetModelStatus](model_server_grpc_api_kfs.md)
-and [REST Model Status](model_server_rest_api_kfs.md)
+It will report in a verbose way all the operations in the mediapipe framework from the graph initialization and also the graphs execution.
+After the model server, you could confirm with the graph has correct format and all the required models are loaded successfully.
+Note that graph loading is not confirming if all the calculators are compiled into the model server build. That can be confirmed with after sending the request to the KServe endpoint.
```

Comment:
```suggestion
Note that graph loading is not confirming if all the calculators are compiled into the model server build. That can be confirmed after sending the request to the KServe endpoint.
```

Is it true? Will the graph load if there's an issue with the calculator? I suppose if the graph loads and gets AVAILABLE, we know that calculators are available.